lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -39,6 +44,8 @@ class ProgressTabView(RetrieveAPIView):
 
         Body consists of the following fields:
 
+        end: (date) end date of the course
+        user_has_passing_grade: (bool) boolean on if the user has a passing grade in the course
```

Comment:
Don't forget writing in the description for has_scheduled_content as well.

Reply:
thanks fixed